src/bidiMapper/domains/log/logHelper.ts

```diff
@@ -92,33 +100,78 @@ export function logMessageFormatter(
  * input: {"type": "object", "value": [["font-size", {"type": "string", "value": "20px"}]]}
  * output: '{"font-size": "20px"}'
  */
-function getSingleRemoteValueText(arg: CommonDataTypes.RemoteValue): string {
+function toJson(arg: CommonDataTypes.RemoteValue): string {
   // arg type validation
-  if (!['number', 'string', 'object'].includes(arg.type)) {
-    throw Error('Invalid value type: ' + arg.toString());
+  if (
+    !['array', 'bigint', 'date', 'number', 'object', 'string'].includes(
+      arg.type
+    )
+  ) {
+    return stringFromArg(arg);
+  }
+
+  if (arg.type === 'bigint' && typeof (arg.value, String)) {
```

Comment:
What is `typeof (arg.value, String)` intended to do? I don’t understand how this can produce anything other than `'function'`. Similar for the other uses of `typeof` below.

Reply:
Good catch! Removed the check here and in other places.